Xamarin.Forms.Core/Items/ItemsUpdatingScrollMode.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+	public enum ItemsUpdatingScrollMode
+	{
+		KeepItemsInView,
```

Comment:
as you happen to use `default(ItemsUpdatingScrollMode)`, you probably have to make sure `KeepItemsInView` is the default and will stay the default.
```suggestion
		KeepItemsInView = 0,
```